src/App.js

```diff
@@ -54,40 +71,68 @@ class App extends Component {
             <Button primary>Primary button</Button>
           </Section>
 
-          <Subtitle>Inputs</Subtitle>
-          <Input />
-
           <Subtitle>Radio</Subtitle>
-          <p>
-            <RadioInput type="radio" id="ice-cream" name="radio-group" defaultChecked />
-            <InlineLabel htmlFor="ice-cream">Ice cream</InlineLabel>
-          </p>
-          <p>
-            <RadioInput type="radio" id="cotton-candy" name="radio-group" />
-            <InlineLabel htmlFor="cotton-candy">Cotton candy</InlineLabel>
-          </p>
-          <p>
-            <RadioInput shadow type="radio" id="lolipop" name="radio-group-2" defaultChecked />
-            <InlineLabel htmlFor="lolipop">Lolipop</InlineLabel>
-          </p>
-          <p>
-            <RadioInput shadow type="radio" id="candy" name="radio-group-2" />
-            <InlineLabel htmlFor="candy">Candy</InlineLabel>
-          </p>
+
+          <InputGroupRow>
+            <InputGroup>
+              <Radio
```

Comment:
I think `InputGroup` is useless. We could only use `InputGroupRow` (named `InputGroup`) and export it like other components.